modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -727,7 +737,7 @@ private[internals] class Renderer(compilationUnit: CompilationUnit) { self =>
               if (recursive) line"$recursive_($struct_" else line"$struct_"
             line"${schemaImplicit}val schema: $Schema_[${product.nameRef}] = $definition"
               .args(renderedFields)
-              .block(line"${product.nameRef}.apply")
+              .block(line"make")
```

Comment:
**nitpick:** I'd try to avoid the block here and pass this in the same line

Reply:
[f0b79af](https://github.com/disneystreaming/smithy4s/pull/1427/commits/f0b79af38cfbf7f131a19c429b77e611589edb83)